containment-rpm-pxe.changes

```diff
@@ -1,4 +0,0 @@
--------------------------------------------------------------------
```

Comment:
Why are removing the changelog and the spec?

Reply:
the package have now a `_service`, so the version, the package and the changes are now synced automatically with `osc service dr`

https://build.opensuse.org/package/show/systemsmanagement:yomi/containment-rpm-pxe

Reply:
the .changes file certainly shouldn't be in git. That's an OBS specialty. Having that in git ist just painful. Using the _service approach is convenient as you get the git history in the package changelog then without having to do anything manually.
Whether or not the spec is in git, well, debatable. I wouldn't put it but there are other opinions too. Alternatively also the _service file can be used to extract it from git by the service:

>     <param name="extract">containment-rpm-pxe.spec</param>